### DISCUSSION TITLE
middleware/rocketmq/best-pratice/singleclustercrosszone/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5596] [Announcements]
# middleware/rocketmq/best-pratice/singleclustercrosszone/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/rocketmq/best-pratice/singleclustercrosszone/

**Comment by cquliaoli** [comment]
图很模糊

**Comment by windsonsea** [comment]
Fixed in PR #5612
